dropwizard-bom/pom.xml

```diff
@@ -88,6 +88,18 @@
                 <artifactId>javax.servlet-api</artifactId>
                 <version>4.0.0</version>
             </dependency>
+            <dependency>
+                <groupId>javax.xml.bind</groupId>
+                <artifactId>jaxb-api</artifactId>
+                <version>2.3.0</version>
+                <scope>runtime</scope>
+            </dependency>
+            <dependency>
+                <groupId>javax.activation</groupId>
+                <artifactId>javax.activation-api</artifactId>
+                <version>1.2.0</version>
+                <scope>runtime</scope>
+            </dependency>
```

Comment:
Does adding the two dependencies above mean that Dropwizard applications will no longer need `--add-modules java.xml.bind` on Java 9+? Right now I need to manually include that option when running on Java 9

Reply:
@RohanNagar Yes, the additional command line parameters for java.xml.bind can be removed.